rosdep/python.yaml

```diff
@@ -6945,6 +6945,16 @@ python3-networkx:
   fedora: [python3-networkx]
   nixos: [python3Packages.networkx]
   ubuntu: [python3-networkx]
+python3-networkx-pip:
```

Comment:
Why are we adding this key at all?  We already have the preferred non-pip key above, so this seems unnecessary.